scripts/filter_advisories.py

```diff
@@ -132,14 +140,16 @@ def main():
     advisories = read_advisories("advisories.json")
 
     # Define the current version to compare against
-    # Set default last run date to the year 2000 if not provided
-    if len(sys.argv) < 3:
-        last_run_date_str = "2000-01-01T00:00:00Z"
+    # Set default last run date to the year 2000 if not provided or is blank
+    if (not sys.argv[2].strip()) or (len(sys.argv) < 3):
```

Comment:
I think this condition needs inverting, otherwise you'll get an index error
```suggestion
    if (len(sys.argv) < 3) or (not sys.argv[2].strip()):
```